### ISSUE TITLE
Why int?

### ISSUE THREAD

**Issue by vmartirosyan** [LevonMuradyan/Alan-Turing-School-Projects#1]
Is there a special need to store each digit in a int value. Why not char or anything shorter than int?

https://github.com/LevonMuradyan/Alan-Turing-School-Projects/blob/791a5a55e59011a17be4aefeb374e62a52feb681/Huge_Integers/src/hugeInt.hpp#L28

**Comment by LevonMuradyan** [comment]
I agree, we can use int8_t or char to store 0-9 numbers, cause the maximum possible digit is 9(0000 1001),, which can be stored in 4 bits, the minimum type size is 1 byte, so the int8_t and char would be better than int.
